app/src/main/java/com/lastaoutdoor/lasta/ui/screen/settings/SettingsScreen.kt

```diff
@@ -39,6 +43,29 @@ fun SettingsScreen(
     navigateBack: () -> Unit,
     signOutAndNavigate: () -> Unit
 ) {
+
+  val showDeleteDialog = remember { mutableStateOf(false) }
+
+  if (showDeleteDialog.value) {
+    AlertDialog(
+        modifier = Modifier.testTag("settingsDeleteDialog"),
+        onDismissRequest = { showDeleteDialog.value = false },
+        title = { Text(text = LocalContext.current.getString(R.string.delete_account)) },
+        text = { Text(text = LocalContext.current.getString(R.string.delete_account_prompt)) },
+        confirmButton = {
+          Button(
+              onClick = {
+                showDeleteDialog.value = false
+                signOutAndNavigate()
+              },
+              modifier = Modifier.testTag("settingsDeleteButton")) {
+                Text(LocalContext.current.getString(R.string.yes))
+              }
+        },
+        dismissButton = { Button(onClick = { showDeleteDialog.value = false }) { Text("No") } },
```

Comment:
There's a hardcoded Text("No") here that should adapt itself following the current language! 
<img src="https://github.com/LASTA-OUTDOOR/lasta/assets/95919886/dd9099f5-c27a-443f-9b63-8f96fff738bb" width="200">

Reply:
Thanks ! good job on spotting it :-)